src/view/com/util/post-embeds/YoutubeEmbed.tsx

```diff
@@ -23,9 +23,9 @@ export const YoutubeEmbed = ({
 
   return (
     <Link
+      asAnchor
```

Comment:
Also noticed that YouTube embeds were rendered as anchors on web, so fixed that

Reply:
Nice!